apps/web/src/routes/UserSettings/UserSettings.js

```diff
@@ -83,10 +84,15 @@ const UserSettings = () => {
       component: <ManageInvitesTab currentUser={currentUser} />
     },
     {
-      name: t('Locale'),
-      path: 'locale',
+      name: t('Language'),
```

Comment:
actually im now wondering if we have a General Settings tab that has language and appearance related stuff. otherwise we have two tabs that each only have 1 or 2 settings on them right now

Reply:
Let's discuss